api/types_test.go

```diff
@@ -0,0 +1,143 @@
+package api
+
+import (
+	"os"
+	"testing"
+)
+
+func TestOIDCConfig_Validate(t *testing.T) {
+	OIDCNotEnabled := false
```

Comment:
Instead of creating the OIDCNotEnabled, OIDCIsEnabled the boolean values can just be set directly in the `fields` struct directly changing the `Enabled` attribute to be a `bool` instead of `*bool`.

Then in the loop you can just pass the address of it, or create a copy in the loop itself and then passing the address to the method call in the loop